spec/models/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../../app/models/user.rb'
+
+describe User do
+  def expect_not_to_save_user_with_no property
+    user = build(:user, property => "")
+    expect(user.save).to be false
+  end
+
+  it 'can be saved to database' do
+    expect{create :user}.to change(User, :count).by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.